PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please try to adhere to the guidelines below as much as possible when submitting your pull request.
+
+- [ ] Please add `[WIP]` to the beginning of your pull request title if you are still working on it or it is not ready for review.
```

Comment:
Rewriting:
```
- [ ] Please prefix the title of your pull request with `[MRG]` if it the contribution is complete and should be subjected to detail review. Otherwise:
- [ ] Incomplete contributions should be prefixed `[WIP]` to indicate a work in progress (and change it to `[MRG]` when it matures). WIPs may act to indicate you are working on something to avoid duplicated work, to request broad review of functionality or API, or to seek collaborators. WIPs often benefit from the inclusion of a [task list](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments) in the PR description.
```